packages/react-native-bpk-component-progress/src/__snapshots__/BpkProgress-test.android.js.snap

```diff
@@ -39,7 +39,7 @@ exports[`Android BpkProgress should not go over the max bound 1`] = `
   style={
     Array [
       Object {
-        "backgroundColor": "rgb(223, 247, 236)",
+        "backgroundColor": undefined,
```

Comment:
🤔

Reply:
Good catch, was using `sagano` instead of `segano` 🤦‍♂

Reply:
And I just realised that was the right name and the token name was wrong :/

Reply:
Oh yeah we fixed that. I think if you get a later version of bpk-tokens it should be ok